test/unit/data-generator.js

```diff
@@ -0,0 +1,361 @@
+var expect = require('chai').expect,
+    objectAssign = require('object-assign'),
+    generate = require('../../lib/data-generator.js'),
+    EventEmitter = require('events').EventEmitter;
+
+var root = {
+      title: ''
+    },
+    suite = {
+      title: 'suite'
+    },
+    test = {
+      title: 'test',
+      ctx: {},
+      err: 'error'
+    },
+    rootSuite = {
+      title: '',
+      indent: 0,
+      tests: []
+    },
+    globalSuite = {
+      title: 'suite',
+      indent: 1,
+      tests: []
+    },
+    nestedSuite = {
+      title: 'suite',
+      indent: 2,
+      tests: []
+    },
+    passTest = {
+      title: 'test',
+      status: true,
+      result: undefined
+    },
+    failTest = {
+      title: 'test',
+      status: false,
+      error: 'error',
+      result: undefined
+    };
+
+/**
+ * Generates a natural strictly positive number
+ *
+ * @returns {Number}
+ */
+function getRandomNumber() {
+  return Math.floor(Math.random() * 100);
```

Comment:
`Math.random` returns a floating point number between `0` and `1`, so this line won't always return a strictly positive number as per the docstring.

I suggest you use [_.random](http://devdocs.io/lodash/index#random) instead.